internal/cobraprompt/prompt.go

```diff
@@ -158,6 +159,11 @@ func (co CobraPrompt) Run(ctx context.Context, root *cobra.Command, cnfg *hazelc
 					exitPromptSafely()
 				}
 			}
+			// clear screen only after sql browser command executed successfully
+			if strings.Trim(in, " ") == "sql" {
+				// lets us invoke ctrl+L shortcut which clears the screen
+				p.Feed([]byte{0xc})
```

Comment:
:thinking: I am surprised that this works in a cross platform way.

Reply:
This byte sequence is directly fed to goprompt's input logic. It does not cross any system specific code AFAIK. Don't think it will create a problem for other systems.

@srknzl Windows is out of scope for now, and we do have many things that will crash on windows. (just to inform about current state)

Reply:
@srknzl It clears the interactive buffer after user exits from SQL Browser. You can observe it by following these steps:
- Enter interactive mode (./hzc)
- Do some operations so that your buffer is not empty (add an element to a map)
- Open the SQL Browser (sql)
- Exit from the SQL Browser (ctrl+Q)
- Observe that your previous operations on the interactive prompt are no longer visible